Shopipy.ApiService/Middleware/BusinessExistsMiddleware.cs

```diff
@@ -0,0 +1,50 @@
+using Microsoft.AspNetCore.Http;
+using Shopipy.BusinessManagement.Services;
+
+namespace Shopipy.Persistence.Data.Middleware
+{
+    public class BusinessExistsMiddleware
```

Comment:
I this this name is more appropriate:
```suggestion
    public class SigmaClass
```

Reply:
Uhm, this is not correct namington.

Reply:
@suetology I definitely agree with the naming, our app is crashing now :(